poker-tournament-visualization/src/app/dev-menu/dev-menu.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit, OnDestroy  } from '@angular/core';
+import { NewPokerGameService } from '../poker-game/new-poker-game.service';
+import { HttpClient } from '@angular/common/http';
+import { devMenuService } from './dev-menu.service';
+
+export interface TimestampJSON {
+  timestamp: number;
+}
+
+@Component({
+  selector: 'dev-menu',
+  templateUrl: './dev-menu.component.html',
+  styleUrls: ['./dev-menu.component.css']
+})
+export class DevMenuComponent implements OnInit, OnDestroy  {
+  public showMenu: boolean = false;
+  public activeJSON: string = "";
+  public timestamp: string = "";
+  public countdown: number = 0;
+  private timerInterval: any;
+  private syncInterval: any;
+
+  public jsonUrl: string = "";
+  public syncUrl: string = "";
+
+  constructor(
+      private newPokerGameService: NewPokerGameService,
+      private devMenuService: devMenuService,
+      private http: HttpClient
+    ) {
+
+  }
+
+  ngOnInit(): void {
+    //auto sync every 10 seconds
+    this.syncInterval = setInterval(() => {
+      this.getRemoteSyncTime();
+    }, 10000);
+  }
+
+  ngOnDestroy() {
+    if (this.timerInterval) {
+      clearInterval(this.timerInterval);
+    }
+    if (this.syncInterval) {
+      clearInterval(this.syncInterval);
+    }
+  }
+
+  loadJsonFile(event: Event):void {
+    const element = event.currentTarget as HTMLInputElement;
+    let fileList: FileList | null = element.files;
+    if (fileList) {
+      if (fileList[0].type == "application/json") {
+        this.newPokerGameService.setNewGameFromJSON(fileList[0]);
+        this.devMenuService.sendLoadGame();
+        this.activeJSON = fileList[0].name;
+      }
+    }
+  }
+
+  loadJsonUrl():void {
+    if (!this.jsonUrl || this.jsonUrl == "") return;
```

Comment:
Igen, der burde ikke være nogen grun til at tjekke for tom streng hvis du alligevel spørger om den er false.
```suggestion
    if (!this.jsonUrl) return;
```